mtp_transaction_uploader/settings.py

```diff
@@ -25,3 +25,6 @@
 API_URL = os.environ.get('API_URL', 'http://localhost:8000')
 
 DS_NEW_FILES_DIR = os.environ.get('DS_NEW_FILES_DIR', '/tmp/ds_new_files')
+
+NOMS_AGENCY_ACCOUNT_NUMBER = os.environ.get('NOMS_AGENCY_ACCOUNT_NUMBER', '11111111')
+NOMS_AGENCY_SORT_CODE = os.environ.get('NOMS_AGENCY_SORT_CODE', '222222')
```

Comment:
you think we should have blank defaults? actually, that could accidentally match anonymous senders